pilot/prompts/development/parse_task.prompt

```diff
@@ -5,6 +5,7 @@ Each step can be either:
 * `command` - command to run (must be able to run on a {{ os }} machine, assume current working directory is project root folder)
 * `save_file` -  create new or update ONE existing file; use this if the existing file is smaller than 20 lines or if many lines need to be changed
 * `modify_file` - update ONE existing file; use this if the existing file is larger than 20 lines and only a few lines need to be updated
+* `human_intervention` - if you need the human to do something, use this type of step and explain in details what you want the human to do.
```

Comment:
you called it `human_intervention` but this prompt is using function call `IMPLEMENT_TASK` which is expecting result `human_intervention_description`. It should be changed here in prompt or in function call + code. I think first option is much easier.

Reply:
good catch - I changed the function call because it's more consistent with other types